### ISSUE TITLE
Menu isn't fullscreen when docked

### ISSUE THREAD

**Issue by vitormmatos** [RSDuck/vba-next-switch#21]
When docked the menu becomes little, close to a quarter of the screen, on top left.
Still the A, B and X labels still appear on right bottom of TV, but with no text, where there is just a black background.

**Comment by dene-** [comment]
We already know and discussing this issue. This issue can be closed now.

**Comment by RSDuck** [comment]
we better keep it open, so everybod

**Comment by fennectech** [comment]
Thanks.  Was about to report this xD    I think the text is there but it is hiding in the black

**Comment by dene-** [comment]
Actually all UI elements positions are hardcoded for 720p screen, they only need to be updated to 1080p.

`
int uiMargins[SOMEMAXSIZE][SOMEMAXSIZE];

typedef enum {
  HANDLED,
  DOCKED
} ScreenMode;

typedef enum {
  marginOne,
  marginTwo,
  paddingOne,
  ...
}

void init() /* or on framebuffer size change, if possible */ {
  uiMargins[HANDLED][marginOne] = someValue;
uiMargins[DOCKED][marginOne] = someOtherValue;
}
`

Probably not the best option but that it might be easier to maintain, to some extent.

**Comment by fennectech** [comment]
Well. I only last week upgraded to a 1080 tv.   Plus the internal panel is 720p so a 1080p signal could cause issues.   If we run at 720p even on 1080p displays that should solve our problem

**Comment by techniponi** [comment]
There's no reason it can't change the resolution based on the display. That's what the rest of the system does - runs at 720p in handheld, and 1080p while docked. It's not locked to one hardcoded resolution, it can change.

**Comment by fennectech** [comment]
So we would be adding profiles for 480p and 720p that are selected based on the display were using?

**Comment by RSDuck** [comment]
no, the "profile" thing is something completely internal. You don't really have to care about it.

**Comment by fennectech** [comment]
What’s your discord username.
